examples/benchmark_problems/simple_tests/cubic-fb.dat

```diff
@@ -0,0 +1,8 @@
+# FitBenchmark Problem
+# This is a simple test problem where we interpolate points that lie exactly on a cubic polynomial.
+software = 'mantid'
```

Comment:
The issue talks about sasview problem sets as well, do you think its worth adding them in the simple tests?

Reply:
I don't know if that's possible at the moment -- I don't know how user functions are defined in SasView (the tutorial just says how to do them in the GUI, as far as I can tell).  Since the SasView/native parsers have now been merged, I'd say this is less of an issue than it was.